07-volumes.md

```diff
@@ -75,12 +80,11 @@ volumes:
     external: true
 ```
 
-### labels
+#### labels
 
 `labels` are used to add metadata to volumes. You can use either an array or a dictionary.
 
-It's recommended that you use reverse-DNS notation to prevent your labels from
-conflicting with those used by other software.
+Use reverse-DNS notation to prevent your labels from conflicting with those used by other software.
```

Comment:
I don't think omitting the "it's recommended" part is good, here. Any other input here is valid, and that could be confusing if we don't call that out. This sentence is explicitly a usage recommendation, not a prescriptive definition.